roles/sidekiq/templates/sidekiq.service.j2

```diff
@@ -0,0 +1,23 @@
+# {{ ansible_managed }}
+
+[Unit]
+Description=sidekiq
+After=syslog.target network.target
+
+[Service]
+Type=notify
+WatchdogSec=10
+
+WorkingDirectory={{ current_path }}
+User={{ unicorn_user }}
+
+ExecStart=/bin/bash -lc 'exec {{ unicorn_home_path }}/.rbenv/shims/bundle exec sidekiq \
```

Comment:
I assume sidekiq reads from `config/sidekiq.yml` by default. Is that it?

Reply:
Yeah, if `config/sidekiq.yml` exists, it'll read it.